scripts/webpack/prod.client.config.js

```diff
@@ -128,10 +128,8 @@ module.exports = ({isDeploy, isStats}) => ({
       append: `\n//# sourceMappingURL=${publicPath}[url]`
     }),
     new InjectManifest({
-      swSrc: 'sw.js',
-      entry: path.join(PROJECT_ROOT, 'packages/client/serviceWorker/sw.ts'),
+      swSrc: path.join(PROJECT_ROOT, 'packages/client/serviceWorker/sw.ts'),
       swDest: 'sw.js',
-      importWorkboxFrom: 'disabled',
```

Comment:
See https://developer.chrome.com/docs/workbox/migration/migrate-from-v4/#build-tool-option-overhaul. Since the `diff` of the `sw.js` file looked right, it seemed safe to remove this setting.